config/locales/cy.yml

```diff
@@ -6,7 +6,7 @@ cy:
           attributes:
             age:
               greater_than_or_equal_to: Rydych yn rhy ifanc i ymuno â phensiwn gweithle. Pan gyrhaeddwch 16 oed gallwch ofyn i’ch cyflogwr eich cofrestru. Os felly, bydd eich cyflogwr yn gwneud cyfraniadau
-              less than: Nid ydych yn gymwys i ymuno â phensiwn gweithle gan eich bod yn hŷn na’r uchafswm oed.
+              less than_or_equal_to: Nid ydych yn gymwys i ymuno â phensiwn gweithle gan eich bod yn hŷn na’r uchafswm oed.
```

Comment:
missing an `_` after `less`

Reply:
i.e. should be `less_than_or_equal_to`

Reply:
♻️ @margOnline 
  - Welsh locale has now been added

♻️ @willhall88 
  - the missing '_' has now been introduced